hal/armv7a/zynq7000/_init.S

```diff
@@ -161,6 +195,32 @@ clear_ttl1:
 	bl _cpy4                                              /* Fill TTL1 with 4 TTL2's addresses; pmap_common.kptab consists of 4 TTL2     */
 
 
+	/* Exceptions vectors and stack TTL1 entry
+	 * map 4MB V 0xffc00000 -> TTL2 of pmap_common.excptab */
+	ldr r0, =(ADDR_TTL1 + (0xffc << 2))                   /* Entry address: TTL1 base address + entry index (0xffc the last 4 entries in TTL1) * 4 bytes (entry size) */
```

Comment:
Put comments within the code (i.e. comment above the line it describes). I see the point in making comment this way, but it causes lines to be way too long

Reply:
Hmm, my intention was to put the general comment above the code and describe each line of assembly code.
I would like to keep this format, my proposition it to break the comment line if it is too long.

Reply:
Look at e.g. line 135 - comment is much closer to the code. At least reduce number of spaces.

Reply:
Done